app/scripts/lib/oauth-errors.js

```diff
@@ -36,13 +50,33 @@ function (_, Errors) {
       errno: 109,
       message: t('Invalid request signature')
     },
+    INVALID_RESPONSE_TYPE: {
+      errno: 110,
+      message: UNEXPECTED_ERROR
+    },
+    UNAUTHORIZED: {
+      errno: 111,
+      message: t('Unauthorized')
+    },
+    FORBIDDEN: {
+      errno: 112,
+      message: t('Forbidden')
+    },
+    INVALID_CONTENT_TYPE: {
+      errno: 113,
+      message: UNEXPECTED_ERROR
+    },
+    INVALID_SCOPES: {
+      errno: 114,
+      message: t('Invalid scopes')
```

Comment:
I've no experience in this regard, but ISTM this could be a fun one for translators.  In this context "scope" is an API term that probably shouldn't be translated.  I wonder if we can re-word the message to make this clearer, e.g. `'Invalid "scope" parameter'`?